src/codeu/chat/server/Controller.java

```diff
@@ -150,6 +174,45 @@ public ConversationHeader newConversation(Uuid id, String title, Uuid owner, Tim
   }
 
   @Override
+  public UserAccessLevel newUserAccessLevel(Uuid conversationId, Uuid userId, AccessLevel accessLevel) {
+
+    final User foundUser = model.userById().first(userId);
+    UserAccessLevel userAccess = null;
+    if(foundUser != null) {
+      userAccess = new UserAccessLevel (userId, accessLevel);
+      model.add(conversationId, userAccess);
+      LOG.info("AccessLevel " + accessLevel + " added to user " + userId);
+    }
+
```

Comment:
Am I correct in understanding that if I try to set the access level for a user who does not exist, then nothing at all will happen?

Ideally, I would want to get some kind of error message telling me that the user I typed doesn't exist. That way, if I made a typo, it would be easier for me to see that I'd made a mistake and harder for me to think that my command had worked when it hadn't.